be/src/common/s3_uri.cpp

```diff
@@ -18,6 +18,8 @@
 #include <fmt/format.h>
 #include <gutil/strings/util.h>
 
+#include <boost/algorithm/string.hpp>
```

Comment:
In StarRocks we don't recommend boost anymore, you can use `strcasecmp` here.

Reply:
fixed